src/nvim/api/vim.c

```diff
@@ -2451,3 +2452,35 @@ void nvim_del_user_command(String name, Error *err)
 {
   nvim_buf_del_user_command(-1, name, err);
 }
+
+/// Get the command line text.
+///
+/// @param[out] err Error details, if any.
+/// @return Command line text.
+String nvim_get_cmdline(Error *err)
+  FUNC_API_SINCE(9)
```

Comment:
Bump this to 10. Also change `NVIM_API_LEVEL` in the main CMakeLists.txt to 10.

Reply:
Done